### PR TITLE
add rust WASM links

### PR DESCRIPTION
**Pull request by efx**
Include 2 standard community links for compiling to WASM from Rust.
https://github.com/mbasso/awesome-wasm

- [x] I've read [CONTRIBUTING.md](https://github.com/mbasso/awesome-wasm/blob/master/CONTRIBUTING.md).
- [x] Description explains the issue / use-case resolved, and auto-closes the related issue(s) (https://help.github.com/articles/closing-issues-via-commit-messages/).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -192,6 +192,8 @@ Please read the [contribution guidelines](CONTRIBUTING.md) if you want to contri
 ### Rust
 - [Rusty Web](https://davidmcneil.github.io/the-rusty-web/)
 - [parity-wasm - WebAssembly interpreter, decoder and encoder in pure Rust](https://github.com/paritytech/parity-wasm)
+- [rustwasm book](https://rustwasm.github.io/docs/book/)
+- [awesome-rust](https://github.com/rust-unofficial/awesome-rust) (*has scattered references to targeting WASM*)
```
[the remark this answers is not in the document]
Comment (mbasso) [reply README.md]: Looks good to me!

File: README.md [hunk 2]
```diff
@@ -192,6 +192,8 @@ Please read the [contribution guidelines](CONTRIBUTING.md) if you want to contri
 ### Rust
 - [Rusty Web](https://davidmcneil.github.io/the-rusty-web/)
 - [parity-wasm - WebAssembly interpreter, decoder and encoder in pure Rust](https://github.com/paritytech/parity-wasm)
+- [rustwasm book](https://rustwasm.github.io/docs/book/)
```
Comment (mbasso) [inline README.md]: Maybe we should put this link in the _Books_ section. What do you think?

Comment (efx) [reply README.md]: @mbasso great idea!

**Comment by mbasso** [comment]
Perfect! Thank you

**Pull request merged**